### DISCUSSION TITLE
Only Remote Write averages of the metrics

### DISCUSSION THREAD

**Discussion by aamargant** [prometheus/prometheus#11467] [Q&A]
Hello, we have an Openshift cluster with Prometheus, we need to remote write some metrics to a Thanos receiver for long term data storage. The question is we have a Grafana instance as well where Prometheus sends all the metrics, with the same Promehteus we want to remote write metrics but we would like to send them once a day for example and send only the averages of the metrics for the day so we send as little data as possible to not use much storage on the Thanos receiver, I have been looking into the documentation but cant seems to find if this is possible or not.

Thanks in advance.

**Comment by bboreham** [comment]
It's not possible in Prometheus; Remote Write sends exactly the samples that were added into the database.

Your description is a little more like Prometheus Federation, which can be configured to pull data at a slower rate than data is stored, but Federation only supplies the latest value each time; you cannot ask for an average.
